eyeshade/workers/reports.js

```diff
@@ -1324,7 +1324,7 @@ exports.workers = {
       if (publisher) {
         query = { publisher: publisher }
         entries = await settlements.find(query)
-        publishers = await mixer(debug, runtime, publisher, range)
+        publishers = await mixer(debug, runtime, publisher && [ publisher ], range)
```

Comment:
aren't we checking if publisher is truthy above? so just `[ publisher ]` would be sufficient?

Reply:
in javascript, the last non-null value is the result, so `publisher && [ publisher ]` is equivalent to `publisher ? [ publisher ] : undefined` (or perhaps `null` or `0`)

Reply:
sure, but as I said we know publisher is truthy because we do this directly above:
```
if (publisher) {
```
so what you wrote here is:
```
true ? [ publisher ] : undefined
```
which obviously makes more sense to write as:
```
[ publisher ]
```

Reply:
at times i must be blind (in this case, not seeing the `if (publisher) {` ... cf. https://github.com/brave-intl/bat-ledger/pull/194